pkg/cmd/issue/view/view.go

```diff
@@ -172,7 +173,7 @@ func printHumanIssuePreview(opts *ViewOptions, issue *api.Issue) error {
 	cs := opts.IO.ColorScheme()
 
 	// Header (Title and State)
-	fmt.Fprintln(out, cs.Bold(issue.Title))
+	fmt.Fprintf(out, "%s #%d", cs.Bold(issue.Title), issue.Number)
```

Comment:
going from Fprintln to Fprintf means we lose the newline:

![image](https://user-images.githubusercontent.com/98482/120394776-8e997900-c2f9-11eb-84d6-658e3098818c.png)

Reply:
Should be fixed, I'm a bit surprised that got through the tests actually 🤔